### PR TITLE
[WIP] Opcodes stuff and documentation

### PR DESCRIPTION
**Pull request by husho**
- Added few opcodes
- Started documenting opcodes (original names used by R* and proper descriptions), because current stuff is outdated incorrect mess, any thoughts on how it should be done correctly?
- Other minor stuff, fixes

- [ ] Opcodes documentation

### CODE REVIEW

File: rwengine/src/script/ScriptMachine.hpp [hunk 1]
```diff
@@ -92,6 +92,9 @@ struct SCMThread {
     unsigned int stackDepth;
     /// Stores the return-addresses for calls.
     std::array<pc_t, SCM_STACK_DEPTH> calls;
+
+    bool deathOrArrestCheck;
```
Comment (ShFil119) [inline rwengine/src/script/ScriptMachine.hpp]: Shouldn't it and other vars be initialized to false?

Comment (husho) [reply rwengine/src/script/ScriptMachine.hpp]: It is done in ScriptMachine::startThread, see ScriptMachine.cpp

Comment (danhedron) [reply rwengine/src/script/ScriptMachine.hpp]: Does this get stored in the save files?

Comment (husho) [reply rwengine/src/script/ScriptMachine.hpp]: Yeah, +0x83 and +0x84 bytes

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 2]
```diff
@@ -7628,12 +7635,22 @@ bool opcode_02bf(const ScriptArguments& args, const ScriptVehicle vehicle) {
...
-    RW_UNUSED(xCoord);
-    RW_UNUSED(yCoord);
-    RW_UNUSED(zCoord);
-    RW_UNUSED(args);
+    /// @todo this is experimental 
+    float closest = 100.f;
+    std::vector<AIGraphNode*> nodes;
+    args.getWorld()->aigraph.gatherExternalNodesNear(coord, closest, nodes);
+    
+    for (auto it = nodes.begin(); it != nodes.end(); it++) {
```
[the remark this answers is not in the document]
Comment (husho) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: Fixed, thanks.

**Pull request closed without merging**